apps/ledger-live-desktop/src/newArch/WalletSync/Flows/Synchronize/01-SynchModeStep.tsx

```diff
@@ -0,0 +1,56 @@
+import { Flex, Icons, Text } from "@ledgerhq/react-ui";
+import React from "react";
+import { useTranslation } from "react-i18next";
+import { Card } from "../../components/Card";
+import styled, { useTheme } from "styled-components";
+
+type Props = {
+  goToQRCode: () => void;
+  goToSynchWithDevice: () => void;
+};
+
+export default function SynchronizeModeStep({ goToQRCode, goToSynchWithDevice }: Props) {
```

Comment:
We usually shortcut Synchronize with `Sync` instead of `Synch`, could you update accordingly, please?